### ISSUE TITLE
Include the collection of the submitted item in Workflow view

### ISSUE THREAD

**Issue by alawvt** [DSpace/dspace-angular#1782]
**Is your feature request related to a problem? Please describe.**
In DSpace 6 XMLUI, the collection is included in the Tasks table, https://demo.dspace.org/xmlui/submissions.
<img width="1071" alt="DSpace6_XMLUI_workflow_tasls" src="https://user-images.githubusercontent.com/13037168/186019374-407ef5a7-2274-423b-828c-a766243319aa.png">
And the full collection hierarchy appears in the breadcrumbs in the Item page, e.g. https://demo.dspace.org/xmlui/handle/10673/77/workflow?workflowID=2.
<img width="1039" alt="DSpace6_XMLUI_workflow_item" src="https://user-images.githubusercontent.com/13037168/186019439-917fc9e5-530b-4756-912b-d6f98f7352df.png">

However, the collection is missing in DSpace 7 Demo, https://demo7.dspace.org/mydspace?configuration=workflow.
<img width="588" alt="DSpace7_workflow_tasks" src="https://user-images.githubusercontent.com/13037168/186019464-d281c851-155f-4ab8-8c8e-bc2a2e54cd1c.png">
The collection is also missing in the item view, e.g. https://demo7.dspace.org/workflowitems/181/view.
<img width="571" alt="DSpace7_workflow_item" src="https://user-images.githubusercontent.com/13037168/186019489-a800c561-c12f-4616-b7fb-ae68af3bbb33.png">

**Describe the solution you'd like**
It is important to include the collection in the approval workflow. The full community/collection hierarchy needs to appear somewhere so the approver can ensure it is going to the right collection. (Repetitive collection names are common in repositories.)  I think the display in DSpace 6 XMLUI is good. 

**Describe alternatives or workarounds you've considered**

**Additional context**

**Comment by tdonohue** [comment]
Copying a comment over here from #1784 as it relates to this ticket.  Breadcrumbs a missing from the workspace/workflow item views.

> Thanks, @tdonohue. #1782  shows the Workflow Item view, where the breadcrumb does not include the collection path.
> <img width="571" alt="DSpace7_workflow_item" src="https://user-images.githubusercontent.com/13037168/186740125-937730af-06e2-47b5-9ca1-4a4fca4edf08.png">
>
>  I have not found other views where the breadcrumb navigation is missing.
>
> _Originally posted by @alawvt in https://github.com/DSpace/dspace-angular/issues/1784#issuecomment-1227149322_

**Comment by tdonohue** [comment]
Agreed, this looks like a usability issue.  Pulling this over to our 7.4 board and flagging as high priority as it seems like a potentially major usability issue to submission & workflow.

**Comment by jtimal** [comment]
Hi

We are working on this view, here is our partial solution
https://files.slack.com/files-pri/T01372XE004-F04LGFESYNB/image.png
